src/components/SingleProductPage/SingleProduct.js

```diff
@@ -0,0 +1,79 @@
+import "./SingleProduct.css";
+import { useParams, useHistory } from "react-router";
+import useProducts from "../../hooks/useProducts";
+import getProductById from "../../utility/getProductById";
+import { FaRegSnowflake, FaArrowLeft } from "react-icons/fa";
+import { BsFillClockFill } from "react-icons/bs";
+
+export default function SingleProduct() {
+  const { singleProductId } = useParams();
+  const products = useProducts();
+  const history = useHistory();
+
+  if (products.length !== 0) {
+    const productInfo = getProductById(Number(singleProductId), products);
+    const {
+      name,
+      image,
+      ingredients,
+      time,
+      refrigeration,
+      goals,
+    } = productInfo;
+
+    const classForTonerImage =
+      name === "Glycolic Acid 7% Toning Solution" ? "toner-image" : "";
```

Comment:
I don't understand. This looks very hacky.

Reply:
:D just says that if the name of the product is Glycolic bla.. the image should have a different class, as the image for that particular product needs to by styled differently to fit into the page's layout

Reply:
But there is only one and only Glycolic bla... product or not?

Reply:
yes, there is only that one

Reply:
I should change it to id really, though. Seems more easily maintainable